Encode and Decode Tree/EncodeAndDecodeTree.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  EncodeAndDecodeTree.swift
+//
+//
+//  Created by Kai Chen on 19/07/2017.
+//
+//
+
+import Foundation
+
+public class TreeNode {
+    public var val: String
+    public var left: TreeNode?
+    public var right: TreeNode?
+
+    public init(_ val: String, left: TreeNode? = nil, right: TreeNode? = nil) {
+        self.val = val
+        self.left = left
+        self.right = right
+    }
+}
+
+public class EncodeAndDecodeTree {
+    private var encodeStr = ""
```

Comment:
FYI, this is a class member variable and violates one of the rules for this problem:

> Do not use class member/global/static variables to store states. Your serialize and deserialize algorithms should be stateless.